pages/common/llm.md

```diff
@@ -19,10 +19,10 @@
 
 `llm install {{package1 package2 ...}}`
 
-- Download and run a prompt against a [m]odel:
+- Download and run a prompt against a model:
 
 `llm --model {{orca-mini-3b-gguf2-q4_0}} "{{What is the capital of France?}}"`
 
-- Have an interactive chat with a specific [m]odel:
+- Have an interactive chat with a specific model:
```

Comment:
```suggestion
- Have an interactive chat with a specific [model]:
```

Reply:
Why would we even have the brackets in the first place?

Reply:
> Why would we even have the brackets in the first place?

Please see this section [Style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#short-option-mnemonics)